src/components/Reports/GoalCalculator/SummaryReport/Steps/PresentingYourGoalStep/PresentingYourGoal.tsx

```diff
@@ -0,0 +1,400 @@
+        title: 'Salary',
+        description:
+          'Salaries are based upon marital status, number of children, tenure with Cru, and adjustments for certain geographic locations.',
+        amount: mockData[0].value,
+      },
+      {
+        title: 'Ministry Expenses',
+        description:
+          'Training, conferences, supplies, evangelism & discipleship materials, communication with ministry partners, ministry travel expenses, etc.',
+        amount: mockData[1].value,
+      },
+      {
+        title: 'Benefits',
+        description:
+          "Includes group medical and dental coverage, life insurance, disability insurance, worker's compensation, and employer contribution to a 403(b) retirement plan.",
+        amount: mockData[2].value,
+      },
+      {
+        title: 'Social Security and Taxes',
+        description:
+          'Since Campus Crusade is a non-profit organization, staff members are responsible for paying the entire amount of Social Security.',
+        amount: mockData[3].value,
+      },
+      {
+        title: 'Voluntary 403b Retirement Plan',
+        description:
+          'Staff members are eligible to contribute to a voluntary retirement program each month.',
+        amount: mockData[4].value,
+      },
+      { title: 'Administrative Charge', amount: mockData[5].value },
+      { title: 'Total Support Goal', amount: total, bold: true },
+      // change amount when real data is added
+      { title: 'Total Solid Support', amount: 2500 },
+    ],
+    [mockData, total, t],
+  );
+
+  return (
+    <Grid container spacing={theme.spacing(6)}>
+      <Box
+        sx={{
+          display: 'flex',
+          justifyContent: 'flex-end',
+          width: '100%',
+        }}
+      >
+        <Button
+          variant="outlined"
+          endIcon={<PrintIcon />}
+          onClick={handlePrint}
+          aria-label={t('Print')}
+        >
+          {t('Print')}
+        </Button>
+      </Box>
+      <Grid ref={printRef} item xs={12}>
+        <Paper
+          sx={{ padding: theme.spacing(3), marginBottom: theme.spacing(3) }}
+        >
+          <Typography sx={{ marginBottom: theme.spacing(2) }} variant="h5">
+            {t('Personal Information')}
+          </Typography>
+
+          <Divider
+            sx={{ margin: `${theme.spacing(2)} ${theme.spacing(-3)}` }}
+          />
+
+          <Table size="small">
+            <TableBody>
+              {personalInfoRows.map((item, index) => (
+                <TableRow key={item.label}>
+                  <StyledTableCell>
+                    <Typography variant="body1" fontWeight="bold">
+                      {item.label}
+                    </Typography>
+                  </StyledTableCell>
+                  <StyledTableCell data-testid="value-typography">
+                    {item.value}
+                  </StyledTableCell>
+                  {index === 0 &&
+                    organizationName?.includes(cruOrganizationName) && (
```

Comment:
`cruOrganizationName` is localized. If the user's language is French, `cruOrganizationName` will be something other than "Campus Crusade for Christ", and this check won't match. I think we should be looking for an unlocalized string here.